htdocs/product/dynamic_price/class/price_parser.class.php

```diff
@@ -131,6 +131,15 @@ public function translatedError()
 	public function parseExpression($product, $expression, $values)
 	{
 		global $user;
+
+		//Check if empty
+		$expression = trim($expression);
+		if (empty($expression))
+		{
+			$this->error = array(20, null);
```

Comment:
What is this ?
this->error shoud contains a string (one error message string or code)
this->errors can contains an array of different error messages